packages/helpers/src/customPropTypes.js

```diff
@@ -37,3 +38,5 @@ export const InputValidTypes = {
 };
 
 InputValidTypes.ALL = Object.values(InputValidTypes);
+
+export const RefOf = PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.any })]);
```

Comment:
```suggestion
export const RefOf = (propType) => PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: propType || PropTypes.any })]);
```

Reply:
This is what you meant, right @alexzherdev ?

Reply:
yeah and `any` can go into the function signature as the default value